rclrs/src/node.rs

```diff
@@ -189,6 +212,27 @@ impl Node {
         Publisher::<T>::new(self, topic, qos)
     }
 
+    /// Creates a [`Service`][1].
+    ///
+    /// [1]: crate::Service
+    // TODO: make service's lifetime depend on node's lifetime
+    pub fn create_service<T, F>(
+        &mut self,
+        topic: &str,
+        callback: F,
+    ) -> Result<Arc<crate::node::service::Service<T>>, RclrsError>
+    where
+        T: rosidl_runtime_rs::Service + 'static,
+        F: Fn(&rmw_request_id_t, &T::Request) -> T::Response + 'static + Send,
```

Comment:
> * FnMut is more general than Fn, which is why it's used in Subscription. But it can't run concurrently with itself, which might come up when implementing executors. For now, I'd favor FnMut for consistency and generality.

If you check the feedback that @mxgrey gave (https://github.com/ros2-rust/ros2_rust/pull/146#discussion_r897549598), I originally had an `FnMut`, but I prefer the change to `Fn` as it makes the code simpler. If we end up with a use case where `FnMut` is necessary, we can always update this.

> * As far as I can see, there's no reason to not give an owned T::Request to the callback – it's strictly more powerful/useful. We also do this in subscriptions.

Done in 40cd43cd